src/install-gleam.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+set -eo pipefail
+
+cd "$RUNNER_TEMP"
+
+VSN=${1}
+FILE_INPUT=gleam-${VSN}-linux-amd64.tar.gz
```

Comment:
Hmm, does this mean 32 bit archs aren't supported? If so, need to note in the README

Reply:
> Hmm, does this mean 32 bit archs aren't supported?

Must be, yes.

I'll note this in the README.

Now for the fun part. When `process.platform === 'win32'`, we use `/^otp_win64_.*.exe$/` for Erlang 😄 

Should we further the note then? And would you be kind to propose a text for it? I don't want to just write "only 64-bit support exists" (especially since this is only important for self-hosted runners)

Reply:
I wouldn't worry about documentation for win32 if win64 builds work on win32.

Reply:
Sure. Can you propose a text and a section to include it in, then?

Reply:
We don't support 32 bit archs for OTP either so I don't think we need to document it.

Reply:
Yeah, and I'm pretty sure GitHub's containers are 64-bit too. I understood what @starbelly wrote as a heads-up for self-hosted runners.

Reply:
Yes GH runs on 64bit.

My understanding is you'd get OTP from setup-beam and given it only supports 64bit, these are the only images it has access to, then making the 32 vs 64 bit distinction pointless.

there is one 32bit platform that is still very important, arm32ve or something like that, but I would cross that bridge when we need to.

+1 from me on keeping things short and simple.

Reply:
> there is one 32bit platform that is still very important, arm32ve or something like that, but I would cross that bridge when we need to.

Agree with this, no need to doc anything then IMO.